sub-packages/bionemo-geneformer/src/bionemo/geneformer/data/singlecell/dataset.py

```diff
@@ -84,7 +83,7 @@ class SingleCellDataset(Dataset):
 
     def __init__(  # noqa: D107
         self,
-        data_path: str | Path,
+        data_path: str,
```

Comment:
is this intentional?

Reply:
My preference would be to allow either. If an internal method only works on one then you could do:

`self.data_path: str = str(data_path)` to make sure whichever a user passes in lands as a str internally.

Reply:
on line 100 currently.